### PR TITLE
Fix a couple issues that came up in the linter's integration test

### PR DESCRIPTION
**Pull request by rictic**
- [x] CHANGELOG.md has been updated

Works around https://github.com/Polymer/polymer-analyzer/issues/623 and also fixes an issue with PackageUrlResolver

### CODE REVIEW

File: src/core/analysis-context.ts [hunk 1]
```diff
@@ -69,9 +68,8 @@ export class AnalysisContext {
     ['json', new JsonParser()],
   ]);
 
-  private readonly _languageAnalyzers = new Map<string, LanguageAnalyzer<any>>([
-    ['ts', new TypeScriptAnalyzer(this)],
-  ]);
+  private readonly _languageAnalyzers =
```
Comment (justinfagnani) [inline src/core/analysis-context.ts]: Can you add a TODO to add back and reference to the issue?

File: src/url-loader/package-url-resolver.ts [hunk 2]
```diff
@@ -39,7 +39,14 @@ export class PackageUrlResolver implements UrlResolver {
 
   canResolve(url: string): boolean {
     const urlObject = parseUrl(url);
-    const pathname = pathlib.normalize(decodeURI(urlObject.pathname || ''));
+    let decodedUrl;
+    try {
+      decodedUrl = decodeURI(urlObject.pathname || '');
+    } catch (e) {
+      return false;
```
Comment (justinfagnani) [inline src/url-loader/package-url-resolver.ts]: this is the kind of thing that makes me like checked exceptions, against all current attitudes...

Comment (rictic) [inline src/url-loader/package-url-resolver.ts]: `Result<string, Error>` is Good and Right, IMO

File: src/core/analysis-context.ts (hunk 1 above)
Comment (rictic) [inline src/core/analysis-context.ts]: Done

**Pull request merged**